modules/rte_mis_school/config/install/field.field.mini_node.school_details.field_school_verification.yml

```diff
@@ -14,7 +14,9 @@ label: 'School Verification'
 description: 'Verify the school application.'
 required: true
 translatable: false
-default_value: {  }
+default_value:
+  -
+    value: 'school_registration_verification_pending'
```

Comment:
If we are hiding the field verification from school edit form, then workflow state `school_registration_verification_submitted` should be used, because then it can't be transitioned to submitted state.

Reply:
Updated!!